server/graphql-schema.js

```diff
@@ -762,15 +766,15 @@ const graphqlSchema = gql`
         Indicates the type of issue. 'CHANGES_REQUESTED' or 'FEEDBACK'.
         'FEEDBACK' is the default type.
         """
-        feedbackType: IssueFeedbackType!
+        feedbackType: String!
```

Comment:
I'd rather moving back to using the `IssueFeedbackType` enum so it's clear what values can be used. 
Unless you had another idea in mind? If so then no problem and the enum can be deleted.